task/backend/analytical_storage.go

```diff
@@ -152,18 +153,45 @@ func (as *AnalyticalStorage) FindRuns(ctx context.Context, filter influxdb.RunFi
 		filterPart = fmt.Sprintf(`|> filter(fn: (r) => r.runID > %q)`, filter.After.String())
 	}
 
+	parsedAfterTime := time.Time{}
+	parsedBeforeTime := time.Now()
```

Comment:
nit: move these inside the `if` block, since they are only referenced within that.